app.py

```diff
@@ -61,8 +61,6 @@ def display_bookmarks():
         comments_count = get_comments_count(COMMENT_PATH, posts)
```

Comment:
https://github.com/mayflowwer/coursework_2_mayflower/pull/1/files#r810525324
```suggestion
        comments_count = functions.get_comments_count(COMMENT_PATH, posts)
```

Reply:
ок, поправила